src/WebJobs.Script/Utility.cs

```diff
@@ -237,13 +236,11 @@ internal static void ApplyBindingData(object value, Dictionary<string, object> b
                 string json = value as string;
                 if (!string.IsNullOrEmpty(json) && Utility.IsJson(json))
                 {
-                    // parse the object skipping any nested objects (binding data
-                    // only includes top level properties)
+                    // parse the object adding top level properties
```

Comment:
The fact that currently this ApplyBindingData is copied code between these repos is unfortunate. This issue is tracked by https://github.com/Azure/azure-webjobs-sdk-extensions/issues/226.